indico/modules/categories/templates/display/root_category.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'categories/display/category.html' %}
+{% from 'categories/display/sidebar.html' import render_news, render_upcoming_events %}
+
+{% block sidebar %}
+    {{ render_news(news) }}
+    {{ render_upcoming_events(upcoming_events) }}
+{% endblock %}
+
+
+{% block category_description %}
+    <p>
+        {% trans -%}
+            Welcome to Indico. Indico allows you to manage complex conferences, workshops and meetings.
```

Comment:
"Indico" twice without anything in between looks strange. maybe we could find some better wording for this whole thing?

cc @pferreir